pyaerocom/aeroval/experiment_output.py

```diff
@@ -78,7 +85,7 @@ class ExperimentOutput(ProjectOutput):
 
     def __init__(self, cfg: EvalSetup):
         self.cfg = cfg
-        super().__init__(cfg.proj_id, cfg.path_manager.json_basedir)
+        super().__init__(cfg.proj_id, f"{cfg.avdb_cfg.resource}:{cfg.path_manager.json_basedir}")
```

Comment:
pyaerocom should not need to know about a json_basedir, which only exists if the json is used as `avdb_cfg.resource=json`.
Database-resources like hire for aerovaldb usually have a one-line resource identifier, typically as uri. There should be no 2 config places for aerovaldb which you have to concatenate, only one.